packages/vscode-extension/src/webview/hooks/useBuildErrorAlert.tsx

```diff
@@ -68,11 +77,20 @@ export function useBuildErrorAlert(shouldDisplayAlert: boolean) {
       'Your project does not have "ios" directory. If this is an Expo project, you may need to run `expo prebuild` to generate missing files, or configure external build source using launch configuration.';
   }
 
+  const isEasBuild =
+    (!!eas?.android && projectState.selectedDevice?.platform === DevicePlatform.Android) ||
+    (!!eas?.ios && projectState.selectedDevice?.platform === DevicePlatform.IOS);
```

Comment:
wouldn't it be easier to pass information whether its eas or not via project? It seems like we have those checks already to tell which build to start, so duplicating it here would just increase the coupling

Reply:
the check if the build should come from eas is very similar to the check here, just 
```
 if (eas?.android) {
    const apkPath = await fetchEasBuild(cancelToken, eas.android, DevicePlatform.Android);
    if (!apkPath) {
      throw new Error("Failed to build Android app using EAS build.");
    }
```

I think it is okay to use Launch Configuration here as we use it on webview side anyway